src/patterns/security-code/default/index.njk

```diff
@@ -0,0 +1,30 @@
+---
+layout: layout-example.njk
+title: Default – Security code
+---
+
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<h1 class="govuk-heading-l">Check your phone</h1>
+
+<p class="govuk-body">We’ve sent you a text message with a security code.</p>
+
+{{ govukInput({
+  label: {
+    text: "Security code"
+  },
+  id: "security-code",
+  name: "security-code",
+  type: "text",
+  autocomplete: "one-time-code",
+  classes: "govuk-input--width-5",
```

Comment:
Notify uses tabular numbers and some extra letter-spacing (`0.04em`) here to make reading the separate numbers slightly easier:
![image](https://user-images.githubusercontent.com/355079/86368720-6894d000-bc75-11ea-94ed-00eea422826b.png)

Relevant commits:
- https://github.com/alphagov/notifications-admin/commit/57069e427df6dd71a0f605c01fbc93e52dd850b0
- https://github.com/alphagov/notifications-admin/commit/a7db2f032a181a366b5093e1c1b3fe8b1bb01b4b

Reply:
Related PR (closed without merging) to add something similar to Frontend:
https://github.com/alphagov/govuk-frontend/pull/1222

We could consider picking that work back up alongside this pattern.

Reply:
👍 like this idea, but I guess it'll have to wait until an extra class gets added to govuk-frontend first.